lib/cfn-nag/custom_rules/CloudfrontMinimumProtocolVersionRule.rb

```diff
@@ -5,7 +5,7 @@
 
 class CloudfrontMinimumProtocolVersionRule < BaseRule
   def rule_text
-    'Cloudfront should use minimum protocol version TLS 1.2'
+    'Cloudfront should use the most advanced available TLS protocol version'
```

Comment:
I think the intent here is that we enforce a secure choice for this parameter, rather than simply "the latest and greatest". Maybe rewording this a bit makes sense.
```suggestion
    'Cloudfront should use a secure minimal protocol version'
```

Reply:
Note that this will also require updating of the spec test language here: https://github.com/stelligent/cfn_nag/blob/master/spec/cfn_nag_integration/cfn_nag_cloudfront_distribution_spec.rb#L29